supported_build_configurations.yaml

```diff
@@ -26,34 +26,33 @@ test-aeat-8800:
 test-actuator-controller:
   - nucleo
   - arm
-  - gimbtonomy
+  - gimbal
   - science
 
 test-blinky:
   - nucleo
   - arm
   - gamepad
-  - gimbtonomy
+  - gimbal
   - pdb
   - science
 
 test-blockingneo:
 #  - nucleo
```

Comment:
this line is why the build is failing. You can't have an empty thing in the yaml

Reply:
We should enforce that test-blockingneo works for nucleo builds, right? @younesr1 @cindyli-13 ?

Reply:
yes

Reply:
what do you mean by "empty thing in the yaml". where's the empty thing?

Reply:
just need to uncomment `nucleo`

Reply:
@younesr1 you can list an app and have no build targets for it. just cuz of the way i wrote the python script lmao. #303 fixes this